cabal-install/Distribution/Client/ProjectPlanning.hs

```diff
@@ -294,48 +294,61 @@ sanityCheckElaboratedPackage ElaboratedConfiguredPackage{..}
 rebuildProjectConfig :: Verbosity
                      -> DistDirLayout
                      -> ProjectConfig
-                     -> IO (ProjectConfig,
-                            [PackageSpecifier UnresolvedSourcePackage])
-rebuildProjectConfig verbosity
-                     distDirLayout@DistDirLayout {
-                       distProjectRootDirectory,
-                       distDirectory,
-                       distProjectCacheFile,
-                       distProjectCacheDirectory
-                     }
-                     cliConfig = do
+                     -> IO
+                          ( ProjectConfig
```

Comment:
IMHO these stylistic changes distract the diff from the meat somewhat, there are some others further down

Reply:
Right, sorry about that. I blanket-applied `brittany` to some stuff and did not revert these unnecessary changes. I will fix that.